projects/storefrontapp-e2e-cypress/cypress/support/general.commands.ts

```diff
@@ -44,3 +61,127 @@ Cypress.Commands.add(
       .click();
   }
 );
+
+export interface LoginUserData {
+  firstName?: string;
+  lastName?: string;
+  alias: string;
+  password?: string;
+  titleCode?: string;
+  username?: string;
+}
+
+export interface LoginOptions {
+  alwaysNew?: boolean;
```

Comment:
This should be only used in development (while debugging cypress and refreshing tests) not to re-create user on every refresh.